nuget/helpers/lib/NuGetUpdater/NuGetProjects/NuGet.CommandLine/AssemblyMetadataExtractor.cs

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Does it make sense to License dependabot-core with two different licenses?

I am no lciense expert, but if `main` uses ``

Reply:
This should've been addressed by 5078f14b92b177eaad0017df6565c4ef8323f9d6.